src/Microsoft.Identity.Client/Platforms/iOS/TokenCacheAccessor.cs

```diff
@@ -206,7 +217,12 @@ private void RemoveAll(string service)
             {
                 foreach (var record in records)
                 {
-                    SecKeyChain.Remove(record);
+                    var result = SecKeyChain.Remove(record);
+
+                    if (result != SecStatusCode.Success)
+                    {
+                        _requestContext.Logger.Warning("Failed to remove cache record from iOS Keychain: SecStatusCode." + result.ToString());
```

Comment:
>"Failed to remove cache record from iOS Keychain: SecStatusCode. [](start = 55, length = 64)

see above... and in this case there is a reason why having the strings in one central location will be beneficial.

I know I have asked about this numerous times. 
We should have a .resx file with all strings in. Even if we do not translate them.

Reply:
@henrik-me I can add a resx file now to get it started